app/routes/v2/select-sbi/index.js

```diff
@@ -38,3 +36,15 @@ module.exports = [{
     }
   }
 }]
+
+const getAllSbis = async (request, error) => {
```

Comment:
This would be be better either as its own Module or as part of the ViewModel. Its just good to keep the routes as clean and light weight as possible